src/library/basetracktablemodel.cpp

```diff
@@ -625,7 +625,10 @@ QVariant BaseTrackTableModel::roleValue(
             VERIFY_OR_DEBUG_ASSERT(rawValue.canConvert<QDateTime>()) {
                 return QVariant();
             }
-            return mixxx::localDateTimeFromUtc(rawValue.toDateTime());
+            if (role == Qt::ToolTipRole || role == kDataExportRole) {
+                return mixxx::localDateTimeFromUtc(rawValue.toDateTime());
```

Comment:
Please use a temporary for mixxx::localDateTimeFromUtc(rawValue.toDateTime()) instead of duplicating the code and return it without any conversion for the tooltip/export roles.